docs/resources/bash.md.erb

```diff
@@ -33,30 +33,42 @@ For example:
 
 <br>
 
-## Matchers
-
-This InSpec audit resource has the following matchers. For a full list of available matchers please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+## Supported Properties
 
-### exist
+* `exit_status`, `stderr`, `stdout`
 
-The `exist` matcher tests if a command may be run on the system:
+<br>
 
-    it { should exist }
+## Property Examples
 
 ### exit_status
 
-The `exit_status` matcher tests the exit status for the command:
+The `exit_status` property tests the exit status for the command:
 
     its('exit_status') { should eq 0 }
 
 ### stderr
 
-The `stderr` matcher tests results of the command as returned in standard error (stderr):
+The `stderr` property tests results of the command as returned in standard error (stderr):
 
     its('stderr') { should eq '' }
 
 ### stdout
 
-The `stdout` matcher tests results of the command as returned in standard output (stdout).
+The `stdout` property tests results of the command as returned in standard output (stdout).
 
     its('stdout') { should match /bin/ }
+
+<br>
+
+## Matchers
+
+For a full list of available matchers please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exist
+
+The `exist` matcher tests if a command may be run on the system:
```

Comment:
Technically, the command may "exist" but not be able to be run due to permissions/mode. I'd change this text to:

```
If an absolute path is provided, the `exist` matcher tests if the command exists on the filesystem at the specified location. Otherwise, the `exist` matcher tests if the command is found in the PATH.
```